modules/apps/site-initializer/site-initializer-cms/src/main/resources/com/liferay/site/initializer/cms/internal/batch/02-object-definition.batch-engine-data.json

```diff
@@ -261,7 +261,7 @@
 				"en_US": "BasicDocuments"
 			},
 			"portlet": false,
-			"restContextPath": "/o/c/basicdocuments",
+			"restContextPath": "/o/c/cms/basic-documents",
```

Comment:
@adolfopa I think you do not need this restContextPath as it will be calculated in the `getModifiableSystemObjectDefinitionRESTContextPath` method.